### PR TITLE
#166790045 : setup react router

### PR DESCRIPTION
**Pull request by shaluchandwani**
#### What does this PR do?
Setup react-router (React-router-dom)

#### Description of Task to be completed?

1. Install and configure the router
creating a dummy route for the login page "/login", the home page  "/" and the Page not found "/inavlid" to test the configuration.
2. Create a components and views file structure where view pages are in the views folder but individual components are in the components folder.

#### How should this be manually tested?

1. Clone the repo, cd into it and npm install to install all the dependencies, 
2.  Then npm run start to start the server, it will automatically open the home page, and when the user types a route that does not exist in the browser it will bring a not found page.

#### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/166790045

#### Screenshots (if appropriate)
Screen Shot 2019-08-14 at 08 47 40
Screen Shot 2019-08-14 at 08 47 21
Screen Shot 2019-08-14 at 08 47 10

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <BrowserRouter>
+    <Switch>
+      <Route exact path="/" component={Home} />
+      <Route path="/login" component={Login} />
+      <Route component={PageNotFound} />
+    </Switch>
+  </BrowserRouter>
+);
+
+export default App;
```
Comment (Joe-Joseph) [inline src/components/App.js:17]: Please add a new line at the end of every file

**Review by Joe-Joseph — APPROVED** [review]
Good job

**Review by kagabof — APPROVED** [review]

**Comment by mnzube** [comment]
Hey Shalu, LGTM except that I think you forgot to make the pivotal tracker story number a link.
Also on the task description try to number or bullet list like the "how should it be manually be tested?'' and begin the sentences with a capital letter for uniformity.

**Review by mnzube — APPROVED** [review]
Good stuff

**Review by pngabo — APPROVED** [review]

**Comment by billkabanga** [comment]
Good work on the PR. @shaluchandwani , However I have noticed that you linked the PR to the wrong PT board story.

**Pull request merged**